scripts/fetch/disch_sites.R

```diff
@@ -0,0 +1,17 @@
+#' Get discharge sites for NWIS
+#'
+#'@param viz the vizlab object 
+fetch.disch_sites <- function(viz){
+  library(dataRetrieval)
+  hucs <- stringr::str_pad(1:21, width = 2, pad = "0")
+  
+  sites <- data.frame(stringsAsFactors = FALSE)
+  for (huc in hucs){
+    resp <- dataRetrieval::whatNWISsites(huc = huc, parameterCd = "00060", outputDataTypeCd = "dv")
+    sites <- rbind(sites, cbind(resp, data.frame(stringsAsFactors = FALSE, huc=huc)))
```

Comment:
if you haven't tried, dplyr's `bind_rows` and `bind_cols` can be noticeably faster. `bind_rows` is smarter too